lang/nl/strings.php

```diff
@@ -58,21 +76,25 @@
     'action' => 'Actie',
     'data' => 'Gegevens',
     'queued' => 'In wachtrij',
-    'last_run' => 'Laatst uitgevoerd',
-    'next_run' => 'Volgende uitvoering',
-    'not_run_yet' => 'Nog niet uitgevoerd',
+    'last_run' => 'Laatst Uitgevoerd',
+    'next_run' => 'Volgende Uitvoering',
+    'not_run_yet' => 'Nog Niet Uitgevoerd',
     'yes' => 'Ja',
     'no' => 'Nee',
     'delete' => 'Verwijderen',
-    '2fa' => 'Tweestapsverificatie',
+    '2fa' => '2FA',
```

Comment:
Write it out to Tweestapsverificatie to be consistent with errors, buttons etc.

Reply:
This isn't supposed to be a translating PR, tho. I specifically left things out UNTRANSLATED to indicate that they AREN'T used.

Reply:
Why should you not remove them if they are not being used?

Reply:
2FA is fine in dutch, tho